app/bundles/LeadBundle/Translations/en_US/messages.ini

```diff
@@ -9,9 +9,13 @@ mautic.campaign.lead.owner="Contact owner"
 mautic.campaign.lead.updatelead="Update contact"
 mautic.campaign.lead.updatecompany="Update contact's primary company"
 mautic.contact.error.notfound="Contact not found."
+mautic.contact.noinformation.devices="Everyone's eager to learn what gadgets your customers love. Make sure our tracking script is part of your website."
+mautic.contact.noinformation.places="Our system is still on the trail. Don’t forget to enable geolocation services in the settings."
```

Comment:
This sill urge users to check geolocation all the time. Is that really a UX improvement? Should we check the setting before we suggest this?

Reply:
Asking if this is a UX improvement is the same as asking why any software in the world isn't bug-free and perfect since v1: **everything is done until someone else have a look and discover it's not** 😄

So let's check this again
- Do you know if there's a ready way for Twig to access this information? like `{{ magicalVariable }}` without a need to modify PHP
- If not, I can just rewrite the message or find another way

Reply:
Try this:
```twig
{% if configGetParameter('ip_lookup_auth') %}
   ...
{% endif %}
```

Reply:
Worked perfectly, thanks!

Where can I see the available parameters (like 'ip_lookup_auth' but for other settings)?

Reply:
In the UI it's in the global config:
![Screenshot 2024-08-27 at 16 13 38](https://github.com/user-attachments/assets/5cd5fc52-8c46-4300-9a86-b57739f963af)
And all the values from the global config form are stored in the `config/local.php` file.

Reply:
Created docs based on your info, thanks!
This will help to improve a lot of other things

- [[UI] Retrieving system information in Twig](https://github.com/mautic/developer-documentation-new/pull/207)